extensions/twitch.js

```diff
@@ -0,0 +1,297 @@
+                defaultValue: false,
+              },
+              bold: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+              size: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "20px",
+              },
+              font: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "MS Comic Sans",
+              },
+            },
+          },
+          {
+            opcode: "image",
+            text: "create image with image [image] with width [width] with height [height] with position [position] is bottom [bottom] is top [top]",
+            blockType: Scratch.BlockType.COMMAND,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+            arguments: {
+              image: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "https://tekinical.github.io/twitchURI.png",
+              },
+              width: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "50px",
+              },
+              height: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "50px",
+              },
+              position: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "absolute",
+              },
+              bottom: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+              top: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+            },
+          },
+          {
+            opcode: "closetext",
+            text: "close text",
+            blockType: Scratch.BlockType.COMMAND,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+          },
+          {
+            opcode: "closeimage",
+            text: "close image",
+            blockType: Scratch.BlockType.COMMAND,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+          },
+          {
+            blockType: Scratch.BlockType.LABEL,
+            text: "Utilities",
+          },
+          {
+            opcode: "true",
+            text: "true",
+            blockType: Scratch.BlockType.BOOLEAN,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+          },
+          {
+            opcode: "false",
+            text: "false",
+            blockType: Scratch.BlockType.BOOLEAN,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+          },
+        ],
+      };
+    }
+
+    setupchat(args) {
+      var Link =
+        "https://nightdev.com/hosted/obschat/?theme=bttv_dark&channel=" +
+        args.username +
+        "&fade=" +
+        args.fade +
+        "&bot_activity=" +
+        args.bots +
+        "&prevent_clipping=" +
+        args.prevent;
```

Comment:
Make sure to cast all of these arguments to a string.

Reply:
How do I exactly do this?

Reply:
Take a look at this https://docs.turbowarp.org/development/extensions/assorted-apis and go to Scratch.Cast

Reply:
![image](https://github.com/TurboWarp/extensions/assets/140775902/06dabb78-183e-43d6-9c10-583e3567665e)
thanks!